scripts/buildPublishedAPI.js

```diff
@@ -101,22 +101,34 @@ async function build() {
   // copy the docs from the current package into the temp dir.
   let packagesDir = path.join(__dirname, '..', 'packages');
   let packages = glob.sync('*/**/package.json', {cwd: packagesDir});
-  for (let p of packages) {
-    let json = JSON.parse(fs.readFileSync(path.join(packagesDir, p), 'utf8'));
-    if (!json.private && json.name !== '@adobe/react-spectrum') {
-      try {
-        // this npm view will fail if the package isn't on npm
-        // otherwise we want to check if there is any version that isn't a nightly
-        let results = JSON.parse(await run('npm', ['view', json.name, 'versions', '--json']));
-        if (results.some(version => !version.includes('nightly'))) {
-          pkg.dependencies[json.name] = 'latest';
-          console.log('added', json.name);
-        }
-      } catch (e) {
-        // continue
-      }
+  async function addDeps() {
```

Comment:
this is the same thing, just all the packages are handled async instead of sync, providing a massive speed boost